### ISSUE TITLE
BadgePositionDirectional

### ISSUE THREAD

**Issue by moga111** [yako-dev/flutter_badges#13]
I suggest adding this option `BadgePositionDirectional` to support devices with RTL support. This would allow specifying `start\end` as alternative to `left\right` which will be handy to position the badge relatively in RTL devices.

Similar to `AlignmentDirectional` as alternative to `Alignment`,
 and `EdgeInsetsDirectional` as alternative to `EdgeInsets`

![image](https://user-images.githubusercontent.com/6555527/67616700-b117f780-f790-11e9-8575-92e617abb41e.png)
RTL

![image](https://user-images.githubusercontent.com/6555527/67616718-f3d9cf80-f790-11e9-871d-6fac374890af.png)
LTR

**Comment by ali2236** [comment]
@yadaniyil replace `Alignment` with `AlignmentGeometry` to support directional alignment.
Also alignment doesn't work for long text:
```Dart
                            child: Badge(
                              alignment: Directionality.of(context)==TextDirection.ltr? Alignment.topRight : Alignment.topLeft,
                              badgeContent: Icon(
                                Icons.error,
                                size: 16.0,
                                color: Colors.red,
                              ),
                              badgeColor: Colors.transparent,
                              elevation: 0,
                              child: Text('Ignore battery optimizations'),
                            ),
```
The location of the badge is always the same

**Comment by RusKryzhanovskiy** [comment]
@moga111, was fixed in 1.1.2
